tests/visitor_actions.test

```diff
@@ -95,18 +95,43 @@ class VisitorActionsTest extends DrupalWebTestCase {
 
     $this->assertTrue(in_array($action_name, array_keys($va_settings['actions'])));
 
-    // Now edit the action and specify '<front>' as the path.
-    $edit = array(
-      'pages' => '<front>',
+    // Create a node and an alias to the node.
+    $node = $this->drupalCreateNode();
+    $node_path = 'node/' . $node->nid;
+    $alias_path = array(
+      'source' => $node_path,
+      'alias' => 'my_node',
     );
-    $this->drupalPost("admin/structure/visitor_actions/manage/$action_name/edit", $edit, t('Save'));
-    $this->assertNoText('The selected pages must contain valid paths.');
-    // Now edit the action and specify an invalid path.
-    $edit = array(
-      'pages' => 'some invalid url',
+    path_save($alias_path);
+
```

Comment:
Hmm, yes I think we should definitely have a test for the language prefix stuff - you might be able to copy some of the set-up from this test in core http://cgit.drupalcode.org/drupal/tree/modules/path/path.test?h=7.x#n263